### PR TITLE
Implement static UI for find product inventory screen (#1wn3apg)

### PR DESCRIPTION
**Pull request by Utkarshkaraiya**
localhost_8100_Product (2)
localhost_8100_Product (4)

### CODE REVIEW

File: src/views/Order.vue [hunk 1]
```diff
@@ -1,118 +1,279 @@
...
+            </ion-item>
+            <ion-item>
+              <ion-label>order created</ion-label>
+              <ion-select value="any">
+                <ion-select-option value="any">any</ion-select-option>
+              </ion-select>
+            </ion-item>
+          </ion-list>
+          <ion-card>
+            <ion-toolbar>
```
Comment (azkyakhan) [inline src/views/Order.vue:76]: Why toolbar is used in ion-card ?

File: src/views/Order.vue [hunk 2]
```diff
@@ -1,118 +1,279 @@
...
-        <div class="order-metatags">
-          <ion-note>Ordered on 7 Jan 2021</ion-note>
-          <ion-badge color="primary">Approved</ion-badge>
-        </div>
+        <div class="order-detail">
+          <div class="order-header">
+            <div class="order-id">
+              <ion-item lines="none">
+                <ion-label>
+                  <h1>Order ID</h1>
```
Comment (azkyakhan) [inline src/views/Order.vue:95]: Just use ion-label here.

File: src/views/Order.vue [hunk 3]
```diff
@@ -1,118 +1,279 @@
...
+              <ion-chip>
+                <ion-icon :icon="pricetag" />
+                <ion-label>Customer Loyalty Status</ion-label>
+              </ion-chip>
+            </div>
 
+            <div class="order-metatags">
+              <ion-note class="metatags">Ordered on 7 Jan 2021</ion-note>
+              <div class="tags">
+                <ion-badge color="primary" slot="end">Approved</ion-badge>
```
Comment (azkyakhan) [inline src/views/Order.vue:114]: Why primary color value is passed over here ?

File: src/views/Order.vue [hunk 4]
```diff
@@ -1,118 +1,279 @@
...
+          </div>
+          <div class="order-items">
+            <ion-card>
+              <ion-item>
+                <ion-thumbnail slot="start">
+                  <Image />
+                </ion-thumbnail>
+                <ion-label>
+                  <p>Brand</p>
+                  <h1>Virtual Name</h1>
```
Comment (azkyakhan) [inline src/views/Order.vue:126]: Just use ion-label.

File: src/views/Order.vue [hunk 5]
```diff
@@ -1,118 +1,279 @@
...
+  },
+  setup() {
+    return {
+      pricetag,
+      ribbon,
+    };
+  },
+};
 </script>
 <style scoped>
```
Comment (azkyakhan) [inline src/views/Order.vue:216]: Improve indentation and place styling in order in which they are used.

File: src/views/Order.vue (hunk 1 above)
Comment (Utkarshkaraiya) [reply src/views/Order.vue:76]: used card header instead of it.

File: src/views/Order.vue (hunk 3 above)
Comment (Utkarshkaraiya) [reply src/views/Order.vue:114]: done

File: src/views/ProductInventory.vue [hunk 6]
```diff
@@ -0,0 +1,342 @@
+<template>
+  <ion-page>
+    <ion-header :translucent="true">
+      <ion-toolbar>
+        <ion-back-button default-href="/" slot="start" />
+        <ion-title>{{ $t("Product inventory") }}</ion-title>
+        <ion-item slot="end" lines="none">
```
Comment (azkyakhan) [inline src/views/ProductInventory.vue]: Why slot end is given to ion-item ?

File: src/views/ProductInventory.vue [hunk 7]
```diff
@@ -0,0 +1,342 @@
...
+        <div class="products">
+          <div class="product">
+            <div class="product-image">
+              <Image/>
+            </div>
+            <div class="mobileonly">
+              <div class="product-detail">
+                <div class="product-virtual">
+                  <ion-item>
+                    <ion-thumbnail slot="start" class="mobileonly-image">
```
Comment (azkyakhan) [inline src/views/ProductInventory.vue:139]: Only use mobile-only class

File: src/views/ProductInventory.vue [hunk 8]
```diff
@@ -0,0 +1,342 @@
...
+                      <Image/>
+                    </ion-thumbnail>
+                    <ion-label>
+                      <p>Brand</p>
+                      <h1>Virtual Name</h1>
```
Comment (azkyakhan) [inline src/views/ProductInventory.vue]: Use ion-label only.

File: src/views/ProductInventory.vue [hunk 9]
```diff
@@ -0,0 +1,342 @@
...
+              </div>
+              <div class="desktop-only">
+                <ion-list>
+                  <ion-list-header>Variants</ion-list-header>
+
+                  <div class="product-detail"> 
+                    <div class="product-virtual">
+                      <ion-item lines="none">
+                        <ion-label>
+                          <ion-label>SKU</ion-label>
```
Comment (azkyakhan) [inline src/views/ProductInventory.vue]: Why ion-label is used inside ion-label ?

File: src/views/ProductInventory.vue [hunk 10]
```diff
@@ -0,0 +1,342 @@
...
+                          <p>{{ $t("Color") }} : color</p>
+                          <p>{{ $t("Size") }}: size</p>
+                        </ion-label>
+                      </ion-item>
+                    </div>
+                    <div class="product-tags">
+                      <ion-chip>
+                        <ion-icon/>
```
Comment (azkyakhan) [inline src/views/ProductInventory.vue]: Why blank ion-icon is used over here ?

File: src/views/Order.vue (hunk 4 above)
Comment (Utkarshkaraiya) [reply src/views/Order.vue:126]: done

File: src/views/ProductInventory.vue (hunk 6 above)
Comment (Utkarshkaraiya) [reply src/views/ProductInventory.vue]: done

File: src/views/ProductInventory.vue (hunk 8 above)
Comment (Utkarshkaraiya) [reply src/views/ProductInventory.vue]: done

File: src/views/ProductInventory.vue (hunk 9 above)
Comment (Utkarshkaraiya) [reply src/views/ProductInventory.vue]: done

File: src/views/ProductInventory.vue (hunk 10 above)
Comment (Utkarshkaraiya) [reply src/views/ProductInventory.vue]: because in Figma instead of the icon there was an image.

File: src/views/ProductInventory.vue (hunk 8 above)
Comment (Utkarshkaraiya) [reply src/views/ProductInventory.vue]: done

File: src/views/ProductInventory.vue (hunk 9 above)
Comment (Utkarshkaraiya) [reply src/views/ProductInventory.vue]: done

File: src/views/ProductInventory.vue [hunk 11]
```diff
@@ -0,0 +1,346 @@
...
+            <ion-list-header>{{ $t("Catalog") }}</ion-list-header>
+            <ion-item>
+              <ion-label>{{ $t("Categories") }}</ion-label>
+              <ion-select value="any">
+                <ion-select-option value="any">all</ion-select-option>
+              </ion-select>
+            </ion-item>
+            <ion-card>
+              <ion-card-header>
+                <ion-title>{{ $t("Purchase date") }}</ion-title>
```
Comment (azkyakhan) [inline src/views/ProductInventory.vue]: You can ion-card-title over here instead of ion-title.

File: src/views/ProductInventory.vue [hunk 12]
```diff
@@ -0,0 +1,346 @@
...
+            </ion-item>
+            <ion-item>
+              <ion-label>{{ $t("Color") }}</ion-label>
+              <ion-select value="any">
+                <ion-select-option value="any">all</ion-select-option>
+              </ion-select>
+            </ion-item>
+            <ion-card>
+              <ion-card-header>
+                <ion-title>{{ $t("Purchase date") }}</ion-title>
```
Comment (azkyakhan) [inline src/views/ProductInventory.vue]: Please refer to ionic framework documentation for using actual UI component.
https://ionicframework.com/docs/api/card

File: src/views/ProductInventory.vue [hunk 13]
```diff
@@ -0,0 +1,346 @@
...
+    IonPage,
+    IonSearchbar,
+    IonSelect,
+    IonSelectOption,
+    IonThumbnail,
+    IonTitle,
+    IonToolbar,
+  },
+
+  setup() {
```
Comment (azkyakhan) [inline src/views/ProductInventory.vue]: Improve indentation

File: src/views/Order.vue (hunk 5 above)
Comment (Utkarshkaraiya) [reply src/views/Order.vue:216]: done

File: src/views/Order.vue (hunk 2 above)
Comment (Utkarshkaraiya) [reply src/views/Order.vue:95]: done

File: src/views/ProductInventory.vue (hunk 13 above)
Comment (Utkarshkaraiya) [reply src/views/ProductInventory.vue]: done

File: src/views/ProductInventory.vue (hunk 12 above)
Comment (Utkarshkaraiya) [reply src/views/ProductInventory.vue]: ok

**Pull request merged**